html/changelogs/Incoming5643-slimechanges.yml

```diff
@@ -0,0 +1,6 @@
+author: Incoming5643
+
+delete-after: True
+
+changes: 
+  - rscadd: Nanotrasen scientists have recently reported mutations within the pink family line of slimes. Station scientists are encoraged to investigate."
```

Comment:
This doesn't actually tell people whats changed; if they wanted to know they'd have to find this PR on github and read it, which rather defeats the point of the changelog entry.